src/test/java/uk/gov/companieshouse/presenter/account/model/PresenterAccountNameTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.companieshouse.presenter.account.model;
+
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import uk.gov.companieshouse.presenter.account.exceptionhandler.ValidationException;
+
+
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
You're not actually mocking anything in here, so you don't need mockito, same in the other model tests